### PR TITLE
Use properties in package props to redirect imports

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.targets [hunk 1]
```diff
@@ -1,14 +0,0 @@
...
-Microsoft.NET.Sdk.Razor.targets
-
-WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
-          created a backup copy.  Incorrect changes to this file will make it
-          impossible to load or build your projects from the command-line or the IDE.
-
-Copyright (c) .NET Foundation. All rights reserved.
-***********************************************************************************************
--->
-<Project ToolsVersion="14.0">
```
Comment (rynowak) [inline src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.targets]: Sorry what's the implication of deleting this file?

File: src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.props [hunk 2]
```diff
@@ -10,9 +10,8 @@ Copyright (c) .NET Foundation. All rights reserved.
 ***********************************************************************************************
 -->
 <Project ToolsVersion="14.0">
-  <Import Project="Sdk.Razor.CurrentVersion.props" />
```
[the remark this answers is not in the document]
Comment (rynowak) [reply src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.props]: "The package"? what does that mean?

[the remark this answers is not in the document]
Comment (rynowak) [reply src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.props]: I really am not sure I understand what's going on

File: src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.targets (hunk 1 above)
Comment (pranavkm) [reply src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.targets]: We no longer import targets or props files from the files NuGet auto-imports. All they do is redirect where the Sdk looks for them. The Sdk will eventually import them.

The specific scenario that removing these files \ imports from the props file that it affects is that if if you reference Microsoft.NET.Razor.Sdk package in a project that does not reference `Microsoft.NET.Razor.Sdk` the Sdk, you no longer get any of the Razor features.

**Pull request merged**